charts/registry-creds/templates/secret.yaml

```diff
@@ -29,7 +29,9 @@ data:
   AWS_SESSION_TOKEN: {{ default "" .Values.ecr.sessionToken | b64enc }}
   aws-account: {{ .Values.ecr.account | b64enc }}
   aws-region: {{ .Values.ecr.region | b64enc }}
-  aws-assume-role: {{ default "" .Values.ecr.assumeRole | b64enc }}
+  {{ if .Values.ecr.assumeRole }}
```

Comment:
I don't understand this change. Tested with and without it the behavior is the same. You left the same default thing in at line 29 so I suspect this is a red herring. Can you try the following commands that work for me wihtout this change:
```
 helm install  --set-string "ecr.accessKeyID=foo" --set-string "ecr.secretAccessKey=foo" registry-creds ./charts/registry-creds/ --set-string ecr.account=132134 --set-string ecr.region=eu-central-1
```

```
 helm install  registry-creds ./charts/registry-creds/
```